README.md

```diff
@@ -49,6 +49,11 @@ Tools can be acquired via instructions in [third_party](/third_party/README.md).
 
 ## Building
 
+Requirements:
+- C++ compiler with C++14 support
+- cmake >= 3.22.0
```

Comment:
We have decided that we'll drop the requirement to `3.8` https://github.com/oneapi-src/unified-runtime/pull/153#discussion_r1063601872

Reply:
Done.